src/bci_build/package/appcontainers.py

```diff
@@ -484,3 +485,41 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+REDIS_CONTAINERS = [
```

Comment:
generally looks good to me. there are a couple of small changes still outstanding:

* please move the REDIS_CONTAINERS definition to a src/bci_build/package/redis.py separate module instead. we're trying to slowly phase out appcontainers.py as a lumping collection of containers and instead have one module per container
* consider creating a README.md.j2 under src/bci_build/package/redis/ with some description and usage instructions (I'm working on something similar for the valkey container, there is an open unfinished PR from me on that
* potentially we have to convert this over to a multistage docker build, but that can be done as a followup, so it is only a NIT for now.